ADOPTERS.adoc

```diff
@@ -0,0 +1,17 @@
+= Adopters
+
+This file is meant to list adopters and users of Kiali. Please feel free to add yourself
+to the respective section.
+
+== Projects
+
+* https://istio.io[Istio]
+* https://rancher.com/docs/rancher/v2.x/en/cluster-admin/tools/istio/[Rancher]
+
+== End-Users
```

Comment:
Even when the information is publicly available, I feel we should make sure to get consents before filing anyone here, no?

Reply:
I guess you are right, that this is better

Reply:
Maybe adding the steps to have your company listed to this section would be a good idea then?

Reply:
https://twitter.com/Rancher_Labs/status/1149329112839753728
https://twitter.com/derberq/status/1149564316066074625

@cfcosta Send a pull-request? :-)  But yes, I was also thinking of adding a blurb at the top about this.